apps/src/templates/progress/ProgressLessonContent.story.jsx

```diff
@@ -13,32 +15,11 @@ export default storybook => {
               "we can send bits and if we want to send a large amount of " +
               "information faster, we have to finds ways to represent the same " +
               "information with fewer bits - we must compress the data."}
-            levels={[
-              {
-                status: 'not_tried',
-                url: '/step1/level1',
-              },
-              {
-                status: 'perfect',
-                url: '/step2/level1',
-              },
-              {
-                status: 'not_tried',
-                url: '/step2/level2',
-              },
-              {
-                status: 'not_tried',
-                url: '/step2/level3',
-              },
-              {
-                status: 'not_tried',
-                url: '/step2/level4',
-              },
-              {
-                status: 'not_tried',
-                url: '/step2/level5',
-              },
-            ]}
+            levels={fakeLevels(5).map((level, index) => ({
+              ...level,
```

Comment:
The URLs will end up being slightly different, but not in a way that really matters.